script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
+
+var a = 10;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier name 'a' is too short (< 2).](https://app.codacy.com/gh/WebHeroSchool/js_test2/pullRequest?prid=5502592)